core/src/main/java/io/questdb/cutlass/http/HttpConstants.java

```diff
@@ -61,6 +61,7 @@ public final class HttpConstants {
     public static final Utf8String URL_PARAM_NM = new Utf8String("nm");
     public static final Utf8String URL_PARAM_O3_MAX_LAG = new Utf8String("o3MaxLag");
     public static final Utf8String URL_PARAM_OVERWRITE = new Utf8String("overwrite");
+    public static final Utf8String URL_PARAM_OWNER = new Utf8String("owner");
```

Comment:
this has to be covered by tests

Reply:
moved it to ent, where it is actually used